app/models/banner.rb

```diff
@@ -6,17 +6,8 @@ class Banner < ApplicationRecord
   scope :active, -> { where(active: true) }
 
   validates_presence_of :name
-  validate :only_one_banner_is_active_per_organization
 
   private
-
-  def only_one_banner_is_active_per_organization
```

Comment:
Even though we will no longer get this validation message in the UI, I think we should still keep it in. For example, a developer could try to update a banner to active in `rails console` and we'd want that to fail if there is already one active.

Reply:
There's also a race condition where when the page loads, there is no active banner, but by the time the form is submitted, another user has set a banner to be active.

Reply:
Ah, thanks, that makes sense. Was on the fence about removing that validation but couldn't think of when it would be necessary. Here's a good reason!